### PR TITLE
fix poll button being visible below voted poll

### PR DESCRIPTION
**Pull request by connyduck**
cc @Tak

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java [hunk 1]
```diff
@@ -215,23 +215,24 @@ private void setTextVisible(boolean expanded,
...
         } else {
+            hidePoll();
             LinkHelper.setClickableMentions(this.content, mentions, listener);
         }
         if (TextUtils.isEmpty(this.content.getText())) {
             this.content.setVisibility(View.GONE);
         } else {
             this.content.setVisibility(View.VISIBLE);
         }
-        setPollVisible(poll != null && expanded);
```
Comment (charlag) [inline app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java]: I'm sorry but I look at this for a few minutes already and I don't see the difference with the old logic

Comment (connyduck) [reply app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java]: The problem with the old logic is that `setPollVisible(true)` was called after the poll was setup, and `setPollVisible(true)` makes the button visible, no matter what.
Now the method is called `hidePoll` and is never called together with `setUpPoll`.

Comment (charlag) [reply app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java]: I see now, thanks, LGTM

Comment (charlag) [reply app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java]: (maybe this explanation should be in commit message :wink: )

**Pull request merged**